src/views/v2/month/event-multiday.php

```diff
@@ -15,25 +15,55 @@
 
 $event    = $this->get( 'event' );
 $event_id = $event->ID;
+$day_number = $this->get( 'day' );
 
 $classes = [ 'tribe-events-calendar-month__event-multiday' ];
+$classes_inner = [ 'tribe-events-calendar-month__event-multiday-inner' ];
 
-if ( $is_featured = tribe( 'tec.featured_events' )->is_featured( $event_id ) ) {
+// Check if it's featured.
+if ( $is_featured = isset( $event->featured ) && $event->featured ) { // @todo: later use tribe( 'tec.featured_events' )->is_featured( $event_id ) or similar
 	$classes[] = 'tribe-events-calendar-month__event-multiday--featured';
 }
+
+// If it starts today and this week, let's add the left border and set the width
+if ( $starts_today = $event->start_date == $day_number ) { // @todo:later we can check mm/dd or even year
```

Comment:
The logic here, at least from description, seems a bit confusing. Maybe I'm not understanding, but the labels we use should also be descriptive enough for users to understand what's happening. To me, we should have one `T/F` flag that is `show`. If `show === true`, we do everything (`start_today` flag, `end_this_week` flag). If `show === false`, we only add `--hidden`.

Reply:
If it's starting today, it means it should be shown. `$starts_today` is a boolean (it can be renamed to `$should_show` or something else) that is defined if the event starting date equals `$day_number` (the day number of the cell)

Reply:
Yes, I think using a `$should_show` variable name would be better and easier to understand.

Reply:
Yes, I didn't put too much focus on that because I feel these will be replaced with template tags, and they will be self descriptive.